src/app/home-page/widget/starred-box/starred-box.component.scss

```diff
@@ -0,0 +1,28 @@
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.list-events {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2; /* number of lines to show */
+  line-clamp: 2;
+  -webkit-box-orient: vertical;
+}
+
+img[alt='User avatar'] {
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  border: solid 1px mat.get-color-from-palette($dockstore-app-gray, 4);
+  margin: 0.8rem;
+}
+
+img[alt='Org avatar'] {
+  width: 6rem;
+}
+
+img[alt='star'] {
+  width: 2rem;
```

Comment:
Would this global style work for that image? https://github.com/dockstore/dockstore-ui2/blob/3df74a1d5c2764a09cb15d5f475e2be8300ad5b7/src/styles.scss#L395